src/crab/type_to_num.cpp

```
@@ -211,6 +249,11 @@ void TypeToNumDomain::forget_type_dependent_values(const Variable type_variable)
             values.havoc(variable_registry.kind_var(kind, type_variable));
         }
     }
+    // Drop every binding in the region just havoced.
+    if (!ptr_sum_bindings.empty()) {
```

Cppcheck (2.21.0)

[performance] Function parameter 'premise_set' should be passed by const reference.

(passedByValue)